docs/release-notes/stable versions.md

```diff
@@ -0,0 +1,31 @@
+---
+id: stable-versions
+title: Mia-Platform stable versions
+sidebar_label: Stable versions
+---
+
+A **stable release** is a version that has been thoroughly tested and is therefore as reliable as possible. It does not include all the latest new features of a beta release.
+
+Versions labeled as _stable_ are particularly suitable for on-premise updates and installations.
+
+Typically, a new stable version is announced every quarter of the year. Therefore, __four stable versions__ will be announced over a one-year period.
+
+For each stable version, a **Maintenance Window** will be granted for a three-months period approximately, pending the next stable version.
+
+Thus, once a [Major/Minor version](/info/version_policy.md) is labeled as stable, patches will be provided throughout the Maintenance Window in case of relevant detected bugs. Then, we recommend regularly upgrading to the latest patch of the stable release.  
```

Comment:
```suggestion
Thus, once a [Major/Minor version](/info/version_policy.md) is labeled as stable, patches on this specific version will be provided throughout the Maintenance Window in case of relevant detected bugs. Then, we recommend regularly upgrading to the latest patch of the stable release.  
```